Form-Controls/index.html

```diff
@@ -7,21 +7,73 @@
   <meta name="description" content="">
   <meta name="viewport" content="width=device-width, initial-scale=1">
   <link rel="stylesheet" href="styles.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet">
 </head>
 <body>
   <header>
     <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <section>
+      <label for="name">Name</label>
+      <input id="name" type="text" minlength="3" maxlength="8" required />
+    </section>
+    <section>
+    <label for="email">Email</label>
+    <input id="email" type="email" required/>
+    </section>
+    <section>
+      <fieldset>
+        <legend>colors</legend>
+      
+        <input id="purple" type="checkbox" name="color" value="purple" />
+        <label for="purple">Purple</label>
+        <br>
+      
+        <input id="green" type="checkbox" name="color" value="green" />
+        <label for="green">Green</label>
+        <br>
+      
+        <input id="pink" type="checkbox" name="color" value="pink" />
+        <label for="pink">Pink</label>
+      </fieldset>
+    </section>
+    <section>
+      <label for="size">Select Size</label>
+<select id="size" name="size">
```

Comment:
Have you installed Prettier?  https://syllabus.codeyourfuture.io/guides/code-style-guide#using-prettier-to-format-code-automatically